quickstart/javascript/node/conversation-transcription/.npmrc

```diff
@@ -1,4 +0,0 @@
-# Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
```

Comment:
These were all added by the SDL security team.
I don't think you can remove these.

Well, technically I think you can reduce them to one .npmrc file in the root of the enlistment, but that won't get rid of the user sign-in prompt when building locally...

Was there some other reason to remove these build config file?

Reply:
aka, if you remove these npmrc files, you will get a CI build failure that the node project is pulling packages from an untrusted package source...

Reply:
The samples in the [JS azure SDK](https://github.com/Azure-Samples/azure-sdk-for-js-samples) manages to present their samples without a .npmrc. Is there any way we can get the same latitude?